### PR TITLE
Feature/docker

### PR DESCRIPTION
**Pull request by Poswark**

### CODE REVIEW

File: api/Properties/launchSettings.json [hunk 1]
```diff
@@ -3,15 +3,14 @@
...
     "iisExpress": {
-      "applicationUrl": "http://localhost:9897",
-      "sslPort": 44350
+      "applicationUrl": "http://localhost:5000;http://host.docker.internal:5000"
     }
   },
   "profiles": {
     "api": {
       "commandName": "Project",
       "launchBrowser": true,
```
Comment (Mteheran) [inline api/Properties/launchSettings.json:12]: Pls don't change the current profile to work with docker by default, create a new profile instead.

File: api/appsettings.Development.json [hunk 2]
```diff
@@ -6,6 +6,6 @@
     }
   },
   "ConnectionStrings": {
-    "DefaultConnection" : "User ID=postgres;Password=postgres;Host=/tmp;Port=5432;Database=dbcolombia;"
```
Comment (Mteheran) [inline api/appsettings.Development.json:9]: Revert this unnecessary change

File: api/appsettings.json [hunk 3]
```diff
@@ -7,6 +7,6 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "DefaultConnection" : "User ID=postgres;Password=postgres;Host=/tmp;Port=5432;Database=dbcolombia;"
```
Comment (Mteheran) [inline api/appsettings.json:10]: Revert this change pls

**Comment by Poswark** [comment]
reviewing
